tests/role_user_test.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""Test auth_role.USER
```

Comment:
This tests pkcli.admin.disable_user. Make pkcli.admin2_test. pkcli.admin_test does too much to the db and doesn't use `fc` (that is it doesn't start a server).

Reply:
[609c03c](https://github.com/radiasoft/sirepo/pull/7070/commits/609c03c32717dcd27067fe26bc7c2bbb4ee16dc1)